website/pages/docs/reference/transformer-spec.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Transformer Spec Reference
+description: Reference for the transformer spec CloudQuery configuration object.
+---
+
+# Transformer Spec Reference
+
+This goes through all the available options for the transformer plugin `spec` object.
+
+## Example
+
+This example configures the `basic` plugin to rename all tables by adding a `cq_` prefix before reaching the destination.
+
+```yaml copy
+kind: transformer
+spec:
+  name: "basic"
+  path: "cloudquery/basic"
+  registry: "cloudquery"
+  version: "VERSION_TRANSFORMER_BASIC"
+
+  spec:
+    transformations:
+      - kind: change_table_names
+        tables: ["*"]
+        new_table_name_template: "cq_{{.OldName}}"
+```
+
+## Spec
+
+### name
+
+(`string`, required)
+
+Name of the plugin. If you have multiple transformer plugins, this must be unique.
+
+The name field may be used to uniquely identify a particular transformer configuration. For example, if you have two configs for the basic plugin for transforming a source table differently in each of two different destination databases, one may be named `basic-1` and the other `basic-2`. In this case, the `path` option below must be used to specify the download path for the plugin.
+
+### registry
+
+(`string`, optional, default: `cloudquery`, available: `github`, `cloudquery`, `local`, `grpc`, `docker`)
+
+- `cloudquery`: CloudQuery will look for and download the plugin from the official CloudQuery registry, and then execute it.
+- `github`: **Deprecated**. CloudQuery will look for and download the plugin from GitHub, and then execute it.
+- `local`: CloudQuery will execute the plugin from a local path.
+- `grpc`: mostly useful in debug mode when plugin is already running in a different terminal, CloudQuery will connect to the gRPC plugin server directly without spawning the process.
+- `docker`: CloudQuery will run the plugin in a Docker container. This is most useful for plugins written in Python, as they do not support the `local`, `github` and `cloudquery` registries.
```

Comment:
Same here `docker` is not supported for this type of plugins yet

Reply:
Removed